data/campaigns/Under_the_Burning_Suns/utils/abilities.cfg

```diff
@@ -1446,7 +1446,7 @@ Marksman attacks are only affected if the chance to hit is greater than 60%."
     [dummy]
         id = taunt
         name = _"taunt"
-        description= _ "Taunts the enemy making them try to attack Kaleh no matter what for a one turn."
+        description= _ "Taunts the enemy making them try to attack Kaleh, no matter what, for one turn."
```

Comment:
Actually it doesn't need the no matter what if you don't mind changing that in this PR.